src/commands/prune.cr

```diff
@@ -8,6 +8,11 @@ module Shards
       def run
         return unless lockfile?
 
+        unless Dir.exists?(Shards.install_path)
+          Log.info { "Pruned nothing, because #{File.basename(Shards.install_path)}/ does not exist" }
```

Comment:
I don't think this should be logged. At least not as `info` severity, which gets printed to STDOUT by default.
If the directory doesn't exist, the command simply is a no-op. No need to be chatty about that.
The same happens when there are no directories in `install_path`.

We can make it a `debug` level log entry:
```suggestion
          Log.debug { "Pruned nothing, because #{File.basename(Shards.install_path)}/ does not exist" }
```
But I'm not sure whether even that is much useful. Removing it entirely is probably fine.